Wikipedia/Code/SearchViewController.swift

```diff
@@ -26,9 +31,6 @@ class SearchViewController: ArticleCollectionViewController, UISearchBarDelegate
             navigationBar.isAdjustingHidingFromContentInsetChangesEnabled = false
             searchBar.becomeFirstResponder()
         }
-
-        /// Terrible hack to make back button text appropriate for iOS 14 - need to set the title on `WMFAppViewController`. For all app tabs, this is set in `viewDidAppear`.
-        parent?.navigationItem.backButtonTitle = title
```

Comment:
I think you need at least this part - I notice from the last Search tab, the root view controller is no longer updating to say "Search". (Launch (lands on Explore) > Search tab > Search for something and drill in > Long press back button > Root context menu item still says "Explore")

Reply:
🤦 I should have really trusted that Matt-of-a-few-weeks-ago knew what he was doing. Thanks, reverted.